index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>CHIRICH PETR</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet"   href="style.css">
+  </head>
+  <body>
+    <div class="name">
+      <h1>Petr Chirich</h1>
+      <img src="accets/images/q38Wh9i8oN4.jpg" alt="ironman" width="400" height="200">
+    </div>
+    <div class="summary">
+      <h3>Summary</h3>
+      <p>My goal is  become as a person who can work in a team 
+      and solve complex problems connected with programming and 
+      web development. I want to improve my English level and 
+      want to learn  logical thinking and upgrade my programming
+      skills with the help of this courses  and later work i hope so. 
+      I learn with enthusiasm and always bring the work to the end.</p>
+    </div>
+    <div class="skills">
+        <h3>Skills</h3>
+      <div class="slills-description">
+        <h4>Languages:</h4>
+        <ul>
+          <li>Pyton beginner;</li>
+          <li>C# beginner;</li>
+          <li>C++ beginner;</li>
+          <li>JavaScript beginner.</li>
+        </ul>
+        <h4>Other technologies:</h4>
+        <ul>
+          <li>HTML;</li>
+          <li>CSS.</li>
+        </ul>
+      </div>
+    </div>
+    <div class="education">
+      <h3>Education</h3>
+      <ul>
+        <li>2004 – 2015 School №11 Soligorsk city</li>
+        <li>2015 – 2020, Belarusian National Technical University
+         <ul>
+            <li>Information Technologies and Robotics faculty</li>
+            <li>Speciality: Industrial robots and robotic complexes</li>
+            <li>Specialization: electrican Engineer</li>
+         </ul>
+        </li>
+      </ul>
+    </div>
+    <div class="experiance"> 
+      <h3>Experience</h3>
+      <p>My main experience is projects on Codecademy and HTMLacademy such 
+         as Fashion Blog, Wine festival Schedule, Architecture Blog and so on.</p>
+    </div>
+    <div class="english-level">
+      <h3>English level</h3>
+      <ol>
+        <li>B1 (Intermediate English)</li>
+        <li>English at school from first  to last class</li>
+        <li>English courses at university for for mounth</li>
+        <li><a href="https://lingualeo.com/ru/profile">Lingualeo</a></li>
+        <li><a href="https://www.duolingo.com/petyachirich">Duolingo</a></li>
+        <li>A bit of practice in France about a month  and about two weeks in Bulgaria</li>
+      </ol> 
+    </div>
+    <div class="contacts">
```

Comment:
for contacts section you may use `address` element:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address